implementations/elixir/ockam/ockam/lib/ockam/transport/tcp/listener/ranch.ex

```diff
@@ -0,0 +1,72 @@
+if Code.ensure_loaded?(:ranch) do
+  defmodule Ockam.Transport.TCP.Listener.Ranch do
+    @moduledoc false
+
+    use Ockam.Worker
+
+    @doc false
+    @impl true
+    def setup(options, state) do
```

Comment:
not sure how `Telemetry` is/isn't injected into modules, but this might also be a good event to track

Reply:
good point, I'll add to Ockam.Worker in a different PR
https://github.com/ockam-network/ockam/blob/mrinal/ockam_hub/implementations/elixir/ockam/ockam/lib/ockam/worker.ex#L61

anything module that does `use Ockam.Worker` already gets Telemetry events on handle_message.
I'll add it for setup as well 👍